common/src/main/java/com/wynntils/models/wynnalphabet/WynnAlphabetModel.java

```diff
@@ -263,7 +264,9 @@ public boolean hasWynnicOrGavellian(String message) {
     }
 
     public boolean containsBrackets(String message) {
-        return (message.contains("[[") && message.contains("]]")) || (message.contains("<<") && message.contains(">>"));
+        Pattern bracketPattern = Pattern.compile("(\\[\\[.*\\]\\])|(<<.*>>)");
+
+        return bracketPattern.matcher(message).matches();
```

Comment:
I think you need to use find(), or you need to add some extra .* around the sides, since afaik matches() will only accept a complete match. Also please test it to verify that I did not inadvertently trick you into writing something that do not work!

Also, extract the Pattern to a constant to save some time compiling it (micro-optimization, but we do it as practice).

Reply:
Ah did a quick test with just the brackets but not with stuff around, will double check

Reply:
Works now. Unless you or anyone else wants to do some more tests it should be good to go

Reply:
Your testing is fine. Thanks!